src/Controller/PrivacyController.php

```diff
@@ -223,6 +225,10 @@ public function privacyAction(Request $request) : Response
 <th>Mar&nbsp;12,&nbsp;2019</th>
 <td>Added explanation of use of anonymised data about how users interact with our sites.</td>
 </tr>
+<tr>
+<th>July&nbsp;24,&nbsp;2019</th>
```

Comment:
Nit pick: previous value was abbreviated month name? `Mar`. Do we care to keep it the same?